dftimewolf/lib/collectors/grr_hosts.py

```diff
@@ -551,7 +621,7 @@ def __init__(self, state):
   def SetUp(self,
             hosts, root_path,
             reason, timeline_format, grr_server_url, grr_username, grr_password,
-            approvers=None, verify=True):
+            approvers=None, verify=True, skip_offline_clients=False):
```

Comment:
linter complains:
```

dftimewolf/lib/collectors/grr_hosts.py:621:2: R0913: Too many arguments (11/10) (too-many-arguments)
```

Reply:
Yeah, unfortunately such is the way of using recipes.